Tasks/AzureFileCopy/AzureFileCopy.ps1

```diff
@@ -66,20 +46,27 @@ $sourcePath = $sourcePath.Trim('"')
 $storageAccount = $storageAccount.Trim()
 
 # azcopy location on automation agent
-$azCopyExeLocation = Get-ToolPath -Name 'AzCopy\AzCopy.exe'
+$azCopyExeLocation = 'AzCopy\AzCopy.exe'
 $azCopyLocation = [System.IO.Path]::GetDirectoryName($azCopyExeLocation)
 
+# Initialize Azure.
+Import-Module $PSScriptRoot\ps_modules\VstsAzureHelpers_
+Initialize-Azure
+
+# Import the loc strings.
+Import-VstsLocStrings -LiteralPath $PSScriptRoot/Task.json
+
 # Import all the dlls and modules which have cmdlets we need
-Import-Module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
-Import-Module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
-Import-Module "Microsoft.TeamFoundation.DistributedTask.Task.Deployment.Internal"
-Import-Module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
+# Import-Module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
+# Import-Module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
+Import-Module "$PSScriptRoot\DeploymentUtilities\Microsoft.TeamFoundation.DistributedTask.Task.Deployment.Internal"
+Import-Module "$PSScriptRoot\DeploymentUtilities\Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs.dll"
 
 # Load all dependent files for execution
 Import-Module ./AzureFileCopyJob.ps1 -Force
 Import-Module ./Utility.ps1 -Force
```

Comment:
Should use $PSScriptRoot rather than assume the working directory. And Files are not modules, So should not use Import-Module here. Import-Module will work here but its not documented way. You can either create module using psm1 file or dotsourcing file .